spec/contracts_spec.rb

```diff
@@ -85,9 +83,20 @@
     end
 
     context "when owner class does not include Contracts" do
+      let(:error) {
+        # NOTE Unable to support this user-friendly error for ruby
```

Comment:
So eigenclasses in jruby and ruby 1.8.7 are really retarded.. They don't have even hierarchy. So it is not possible to have `Contract` method without calling `include Contracts` on owner of eigenclass.

/cc @egonSchiele 

WDYT, is it okayish fix for spec failure or not.

Basically what changes between normal versions and these ones is that user will not get nice error when he forgot to include `Contracts` in owner class of eigenclass. He will just get `NoMethodError`.